web/src/components/BackupRestore.jsx

```diff
@@ -239,7 +239,6 @@ class BackupRestore extends React.Component {
         renderNotVeleroMessage={this.renderNotVeleroMessage}
         hideCheckVeleroButton={this.state.hideCheckVeleroButton}
         isLicenseUpload={true}
-        history={this.props.history}
```

Comment:
This is the only `withRouter` prop used in this component. `ConfigureSnapshots` doesn't use it so we can safely delete it.